Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN cd /home/jenkins/bin && \
   curl -LJ https://github.com/aquasecurity/tfsec/releases/download/v0.56.0/tfsec-linux-amd64 -o tfsec && \
   chmod 755 tfsec
 
-RUN curl -L "$(curl -s https://api.github.com/repos/tenable/terrascan/releases/latest | grep -o -E "https://.+?_Darwin_x86_64.tar.gz")" > terrascan.tar.gz && \
+RUN curl -L "$(curl -s https://api.github.com/repos/tenable/terrascan/releases/latest | grep -o -E "https://.+?_Linux_x86_64.tar.gz")" > terrascan.tar.gz && \
```

Comment:
I verified that I can get the Terrascan version successfully
![image](https://user-images.githubusercontent.com/82825597/191799412-07ab36b2-941f-4ac3-a7ae-a63d136fc6a5.png)